definitions/tools/concordance.cwl

```diff
@@ -1,48 +1,44 @@
+#!/usr/bin/env cwl-runner
+
 cwlVersion: v1.0
 class: CommandLineTool
-label: "Concordance tool"
-baseCommand: ["python", "/opt/concordance/newConcordance.py"]
+label: "Concordance checking between Tumor and Normal BAM"
+baseCommand: ["somalier"]
 requirements:
-    - class: DockerRequirement
-      dockerPull: "mgibio/concordance"
     - class: ResourceRequirement
-      ramMin: 4000
+      coresMin: 1
+      ramMin: 8000
+      tmpdirMin: 10000
+    - class: DockerRequirement
+      dockerPull: "brentp/somalier"
```

Comment:
Is there a versioned docker image available?  Relying on `latest` can lead to trouble later.

Reply:
Added the tag **dockerPull: "brentp/somalier:v0.1.5"** and did the push